terraform/shared/video-player/index.html

```diff
@@ -77,8 +74,8 @@ <h2 id="loadingTitle">Locating stream...</h2>
         videojs.Hls.xhr.beforeRequest = function(options) {
           // Replace IPFS gateway of origin with that of this node
           options.uri = options.uri.replace(ipfs_gateway_origin, ipfs_gateway_self);
-          if (options.uri.indexOf("ipfs")) {
-            document.getElementById("loadingTitle").innerHTML="Locating stream via IPNS..."
+          if (options.uri.indexOf("/ipfs/")) {
+            document.getElementById("loadingTitle").innerHTML="Located stream via IPNS..."

```

Comment:
Corrected false matching earlier. If it tries to download an /IPFS/ link then it must have found the ipns stream successfully.  So message is it LOCATED it.

before it was matching against ipfs-server <- so it would incorrectly report it found it.